bims/api_views/location_site.py

```diff
@@ -289,6 +294,138 @@ def get(self, request):
             return Response(response_data)
 
 
+    def get_biodiversity_data(self, collections):
+        biodiversity_data = {}
+        taxa = self.get_origin_cons_endemsim_data(collections)
+        # If I found more than one class of animal
+        if 'Actinopterygii' not in taxa:
+            print('Hey there, you are not a fish!')
```

Comment:
And you should never print in django - it will raise  500 in production mode

See https://stackoverflow.com/questions/3182298/where-do-things-go-when-i-print-them-from-my-django-app

Using logging rather....though in the light of the bad smell comment above we should not even be in ths code path...

Reply:
Forgot to remove this - and yes after I fix the hardcoded key this code path shouldn't exist.